libs/features/georoutingv2/georoutingv2.js

```diff
@@ -110,14 +113,18 @@ function getGeoroutingOverride() {
   return hideGeorouting === 'on';
 }
 
-function decorateForOnLinkClick(link, prefix) {
+function decorateForOnLinkClick(link, prefix, currPrefix) {
   link.addEventListener('click', () => {
     const modPrefix = prefix || 'us';
+    const modCurrPrefix = currPrefix || 'us';
```

Comment:
Since this is used only if the condition on L125 is met, maybe it can be moved inside that `if`

Reply:
done